code/modules/language/felinid.dm

```diff
@@ -1,13 +0,0 @@
-/datum/language/felinid
-	name = "Felinid"
-	desc = "A language devised by felinids to secretly plot against regular humans"
-	exclaim_verb = "nyas"
-	key = "f"
-	flags = TONGUELESS_SPEECH
-	space_chance = 90
-	syllables = list(
-		"meow", "nya", "nyam","mm","mrow","mmrrrr"
```

Comment:
"meow", "mow", "rrrmow","mm","mrow","mmrrrr"

Reply:
"mm" doesn't make a lot of sense as a ""cat"" language thing tbh